src/content/docs/en/community-resources/content.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Community Educational Content
+description: Discover community-produced tutorials, guides, articles and videos to help you learn and build with Astro!
+i18nReady: false
```

Comment:
Note that while `talks.mdx` is `i18nReady: true`, this one is false, in case that's not on purpose.

Reply:
Thanks, Yan, I'll change it!